citus-enterprise.spec

```
@@ -1,20 +1,19 @@
-%global pgmajorversion 11
-%global pgpackageversion 11
+%global pgmajorversion 10
+%global pgpackageversion 10
 %global pginstdir /usr/pgsql-%{pgpackageversion}
 %global sname citus-enterprise
 %global pname citus
-%global debug_package %{nil}
 %global unencrypted_package "%{getenv:UNENCRYPTED_PACKAGE}"
 
 Summary:	PostgreSQL-based distributed RDBMS
 Name:		%{sname}%{?pkginfix}_%{pgmajorversion}
 Provides:	citus_%{pgmajorversion}
 Conflicts:	citus_%{pgmajorversion}
-Version:	9.0.3.citus
+Version:	8.3.4.citus
 Release:	1%{dist}
-License:	Commercial
+License:	AGPLv3
```

I do not think `AGPLv3` is a wise choice here.

Let's keep it as `Commercial` please.

```suggestion
License:	Commercial
```